ibm_db2/tests/metrics.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+STANDARD = [
```

Comment:
Could we use the list in the integrations instead of duplicating it here. That way we could check that we are not missing things.

Reply:
These are lists of metrics, not columns